memory_reference/include/mem_ref_detail.hpp

```diff
@@ -21,17 +21,6 @@ struct Mem_Ref_Detail<CUDAGPU>{
 template <class T>
 using Mem_Ref = typename Mem_Ref_Detail<T>::type;
 
-std::vector<std::string> get_available_device()
```

Comment:
Please bring back the get_available_device() function. It makes easier to develop algorithm. Also it should return an `std::tuple<string, bool>`. The string names the backend and the bool show, if is active.